packages/p2p/src/components/advertiser-page/advertiser-page.scss

```diff
@@ -0,0 +1,99 @@
+.advertiser-page {
+    &__container {
+        display: flex;
+        flex-flow: column;
+        height: calc(100vh - 47px - 53px - 60px - 52px - 36px - 2.4rem - 1.6rem);
```

Comment:
pls add comment as to what are these amounts

Reply:
I actually meant to change this, this has been removed now. Thank you so much for reviewing this! 😅